File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/sqlserver/IntegerTypes.scala
package dev.typr.foundations.docs.sqlserver
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*



@SuppressWarnings(Array("unused"))
object IntegerTypes:
  //start
  val tinyType: SqlServerType[Uint1] = SqlServerTypes.tinyint   // Note: unsigned!
  val intType: SqlServerType[Int] = SqlServerTypes.int_
  val bigType: SqlServerType[Long] = SqlServerTypes.bigint
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/sqlserver/IntegerTypes.java
package dev.typr.foundations.docs.sqlserver;

import dev.typr.foundations.SqlServerType;
import dev.typr.foundations.SqlServerTypes;
import dev.typr.foundations.data.Uint1;

@SuppressWarnings("unused")
public class IntegerTypes {
  // start
  SqlServerType<Uint1> tinyType = SqlServerTypes.tinyint; // Note: unsigned!
  SqlServerType<Integer> intType = SqlServerTypes.int_;
  SqlServerType<Long> bigType = SqlServerTypes.bigint;
  // stop
}
